### PR TITLE
Update references to AddressBook

### PR DESCRIPTION
**Pull request by akhilchainani**

### CODE REVIEW

File: packages-ts/gauntlet-terra-contracts/src/lib/middlewares.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
-import { TerraCommand, AddressBook, logger } from '@chainlink/gauntlet-terra'
+import { TerraCommand, logger } from '@chainlink/gauntlet-terra'
 import { Middleware, Next } from '@chainlink/gauntlet-core'
+import { AddressBook } from '@chainlink/gauntlet-core/dist/commands/addressBook'
```
Comment (RodrigoAD) [inline packages-ts/gauntlet-terra-contracts/src/lib/middlewares.ts]: Let's make the change on `gauntlet-core` first, make the release, and then update `gauntlet-terra`

File: packages-ts/gauntlet-terra/src/commands/logger.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { logger } from '@chainlink/gauntlet-core/dist/utils'
-import { AddressBook } from './addressBook'
+import { AddressBook } from '@chainlink/gauntlet-core'
 import { assertions } from '@chainlink/gauntlet-core/dist/utils'
 
 type COLOR = 'red' | 'green' | 'blue' | 'yellow' | 'cyan' | 'magenta'
```
Comment (RodrigoAD) [inline packages-ts/gauntlet-terra/src/commands/logger.ts]: The `TerraLogger` can be replaced by the new `Logger` from core right?

Comment (akhilchainani) [reply packages-ts/gauntlet-terra/src/commands/logger.ts]: yeah there's another PR open to address that - I still need to update that.

Comment (RodrigoAD) [reply packages-ts/gauntlet-terra/src/commands/logger.ts]: Let's include both changes in this PR

Comment (akhilchainani) [reply packages-ts/gauntlet-terra/src/commands/logger.ts]: cool I just rebased it with the new changes! So this should be good now @RodrigoAD

**Review by RodrigoAD — APPROVED** [review]
Looks good!

**Pull request merged**